contrib/gitian/gitian-osx.yml

```diff
@@ -93,7 +93,7 @@ script: |
   # Build dependencies for each host
   export TAR_OPTIONS=--mtime=2000-01-01T12:00:00
   for i in $HOSTS; do
-    make ${MAKEOPTS} -C ${BASEPREFIX} HOST="${i}"
+    make ${MAKEOPTS} -C ${BASEPREFIX} HOST="${i}" GITIAN=1
```

Comment:
Ideally it should only be `GITIAN=1` if it gets launched from `gitian-build.py`.

There is a (work in progress) dockerfile that takes `gitian-*.yml` files as input and that also uses ccache, if `GITIAN=1` is hardcoded it would lose ccache functionality.

Reply:
Is that also using the gbuild script from https://github.com/devrandom/gitian-builder.git or is it using something else?

Reply:
OK, if it's not using gbuild then I can switch on one of gbuild's env vars instead.